src/index.html

```diff
@@ -6,8 +6,63 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700;900&family=Roboto:wght@100;300;400;500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700;900&family=Roboto:wght@100;300;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div data-qa="card" class="card">
+
+      <img
+       class="card__image"
+       src="/src/images/imac.jpeg"
+       alt="Image"
+       >
+
+      <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina
+        <br>
+        5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__id">
+        Product code: 195434
+      </p>
+
+      <div class="card__revard">
+
+        <div class="card__stars">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
+        <p class="card__review">Reviews: 5</p>
+
+      </div>
+
+      <p class="card__price">
+        <span class="card__price-label">Price:</span>
+        <span class="card__price-number">$2,199</span>
+      </p>
+
+      <a
+       href="#"
+       data-qa="hover"
+       class="card__buy"
+
+       >
```

Comment:
```suggestion
      <a
         href="#"
         data-qa="hover"
         class="card__buy"
       >
```

Use such a style for tags which has 3 or more attributes